src/main/java/duke/commands/DeadlineCommand.java

```diff
@@ -0,0 +1,35 @@
+package duke.commands;
+
+import static duke.ui.Ui.LS;
+
+import duke.storage.Storage;
+import duke.tasks.Deadline;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Deadline command to handle Deadline task logic.
+ */
+public class DeadlineCommand extends Command {
+    private String desc;
+    private String by;
+
+    /**
+     * Constructor to create DeadlineCommand object.
+     */
+    public DeadlineCommand(String desc, String by) {
+        this.desc = desc;
+        this.by = by;
+    }
```

Comment:
Likewise, you can consider inserting a newline between the functions for readability and consistency across all your code (All your command classes.)

Reply:
Ok will take note!